content/docs/fundamentals/converting/resolving-errors@ko.md

```diff
@@ -0,0 +1,350 @@
+
+{{ image('/static/img/docs/tutorials/tut-convert-html-not-responsive.png', 412, 660, align='center third', caption='이미지가 반응하지 않습니다') }}
+
+AMP는 사용자가 지정한 너비와 높이에 따라 해당 요소의 가로 세로 비율을 계산할 수 있습니다.  이를 통해 AMP 레이아웃 시스템에서는 다양한 방법으로 요소를 배치하고 크기를 조정합니다.  `layout` 속성은 AMP에 사용자가 요소를 어떻게 배치하고 늘리고 싶어 하는지 알려줍니다.
+
+이미지를 늘리고 크기를 조정할 수 있도록 레이아웃 속성을 `responsive`로 **설정**해 봅시다.
+
+```html
+<amp-img src="mountains.jpg" layout="responsive" width="266" height="150"></amp-img>
+```
+
+완료되었습니다. 이미지의 가로 세로 비율이 알맞게 변경되었으며 화면 너비에 맞춰 화면을 채우고 있습니다.
+
+{{ image('/static/img/docs/tutorials/tut-convert-html-responsive.png', 412, 660, align='center third', caption='이미지가 반응합니다!') }}
+
+{% call callout('읽어보기', type='read') %}
+[AMP 레이아웃 사양](/ko/docs/design/amp-html-layout.html)에서 AMP 레이아웃 시스템에 관해 자세하게 알아보세요.
+{% endcall %}
+
+## 성공입니다.
+
+이제 AMP 문서가 다음과 같이 작성되어 있어야 합니다.
+
+```html
+<!doctype html>
+<html ⚡ lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+    <link rel="canonical" href="/article.html">
+    <link rel="shortcut icon" href="amp_favicon.png">
+
+    <title>News Article</title>
+
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      body {
+        width: auto;
+        margin: 0;
+        padding: 0;
+      }
+
+      header {
+        background: Tomato;
+        color: white;
+        font-size: 2em;
+        text-align: center;
+      }
+
+      h1 {
+        margin: 0;
+        padding: 0.5em;
+        background: white;
+        box-shadow: 0px 3px 5px grey;
+      }
+
+      p {
+        padding: 0.5em;
+        margin: 0.5em;
+      }
+    </style>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+  </head>
+  <body>
+    <header>
+      News Site
+    </header>
+    <article>
+      <h1>Article Name</h1>
+
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam egestas tortor sapien, non tristique ligula accumsan eu.</p>
+
+      <amp-img src="mountains.jpg" layout="responsive" width="266" height="150"></amp-img>
+    </article>
+  </body>
+</html>
+```
+
+페이지를 새로고침하고 콘솔 출력을 확인합니다. 다음 메시지가 표시되어야 합니다.
+
+<pre class="success-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).